capstone/src/main/webapp/dashboard.html

```diff
@@ -13,9 +13,59 @@
     <!--Let browser know website is optimized for mobile-->
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   </head>
-  <body>
-     <h3>Dahsboard<h3>
+  <body onload="getCurrentUserInfo()">
+     <nav class="nav-wrapper">
+        <a href="" class="brand-logo">Studentia</a>
+        <a href="#" class="sidenav-trigger" data-target="slide_out" ><i class="material-icons">menu</i></a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="findStudySet.html">Search</a></li>
+          <li><a href="/">Home</a></li>
+        </ul>
+    </nav>
+
+    <ul class="sidenav" id="slide_out">
+          <li><a href="findStudySet.html">Search</a></li>
+          <li><a href="index.html">Home</a></li>
+    </ul>
+
+
+    
+    <div class="container">
+        <blockquote class="flow-text">Welcome to your personal dashboard, start learning !! </blockquote>
+        <div class="row">
+            <div class="col s12 m6 l6 responsive-spaced-box z-depth-5">
+                    <h4>Your Profile Details</h4>
+                    <hr>
+                    <p>Full Name: <span id="full_name"></span></p>
```

Comment:
Use hyphens for IDs rather than underscores: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters